### PR TITLE
Introduction to Kernel Methods in Machine Learning

### LINKED ISSUE #4567: Introduction to Kernel Methods in Machine Learning
#### Proposed article introduction
Kernel method in machine learning is defined as a class of pattern analysis algorithms that are used to study and find general types of relations (such as correlation, classification, ranking, clusters, principle components, and so on) in datasets by explicitly transforming raw data representation into feature vector representation using a user-specified feature map, allowing the high dimensional implicit feature space of these data to be operated with computer systems.

#### Key takeaways 
Principle components analysis
Support vector machine
spectral clustering

#### Article quality
How is your article unique? Tell us what makes your approach different from similar articles that have published on the same topic?
Is yours more in-depth? Does it cover additional topics? Do you provide handy tips or anecdotal advice?

#### References
N/A
### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by Brian3811**
### Article checklist

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #4567

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by Brian3811** [comment]
This closes #4567

**Comment by Brian3811** [comment]
Hello @paulodhiambo this is an update of PR #5309

File: content/articles/introduction-to-kernel-methods-in-machine-learning/Index.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
+# Introduction to Kernel Methods in Machine Learning
+
+### Prerequisites
+- The Reader should have basic knowledge in machine learning [Click here](https://www.digitalocean.com/community/tutorials/an-introduction-to-machine-learning) to read more about machine learning basics.
+- The reader should be able to understand basic terms like Support Vector Machine, classifications, unsupervised learning and supervised learning.
```
Comment (paulodhiambo) [inline content/articles/introduction-to-kernel-methods-in-machine-learning/Index.md:5]: reword to "- The reader should understand basic terms like Support Vector Machine, classifications, unsupervised learning, and supervised learning."

File: content/articles/introduction-to-kernel-methods-in-machine-learning/Index.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+- Before digging deeper into the operation of Kernel Methods, it is necessary to first understand support vector machines, or SVMs, because kernels are implemented in SVM models.
+
+
+## Introduction
+The concept of "data drawn from some probability distribution" underpins many techniques in statistics and machine learning. A probability distribution is a model for the frequency of data points, i.e., how likely it is to see a particular value, outcome, or other occurrence. The parameters of the underlying distribution are used to define parametric methods (or algorithms).
+
+The assumed model for the likelihood, however, may not be correct; the "actual" distribution may be skewed, multimodal, with different components, and so on. In general, the scientist analyzing the data is unaware of this. This article aims to demonstrate the fundamental concepts underlying kernels as well as their applications in statistics and machine learning.
```
Comment (paulodhiambo) [inline content/articles/introduction-to-kernel-methods-in-machine-learning/Index.md:12]: reword to "However, the assumed model for the likelihood may not be correct; the "actual" distribution may be skewed, multimodal, with different components, and so on. In general, the scientist analyzing the data is unaware of this. This article aims to demonstrate the fundamental concepts underlying kernels and their applications in statistics and machine learning."

File: content/articles/introduction-to-kernel-methods-in-machine-learning/Index.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+
+
+### Tables of Content
+- [What are kernels?](#what-are-kernels?)
+- [Supervised and unsupervised machine learning](#supervised-and-unsupervised-machine-learning)
+- [Kernel methods in machine learning](#kernel-methods-in-machine-learning)
+- [Conclusion](#conclusion)
+
+### _What are kernels?_
+Kernels, <span style="color:blue">also known as kernel techniques or kernel functions, are a collection of distinct forms of pattern analysis algorithms.</span> Using a linear classifier, they are utilized to solve a non-linear problem. SVM (Support Vector Machines) use Kernels Methods to solve classification and regression issues. The SVM (Support Vector Machine) employs a technique known as the ["Kernel Trick"](https://towardsdatascience.com/the-kernel-trick-c98cdbcaeb3f) in which the data is processed and an optimal boundary for the various outputs is determined.
```
Comment (paulodhiambo) [inline content/articles/introduction-to-kernel-methods-in-machine-learning/Index.md:22]: be consistent, either use non-linear or nonlinear throughout the article.

**Review by paulodhiambo — CHANGES_REQUESTED** [review]
@Brian3811 make the requested changes then I'll proceed with the tech review

**Comment by Brian3811** [comment]
Hello @paulodhiambo the requested changes are in place

**Comment by hectorkambow** [comment]
@Brian3811 lets be sure to go through the entire article to catch any issues that may have not been caught by our peer reviewer. 

>Our Peer Reviewer and Content Moderator teams do NOT provide any revisions services. All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner.
ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.

cc @paulodhiambo be sure to keep this mind as we go through the changes and tech review 👍

File: content/articles/introduction-to-kernel-methods-in-machine-learning/Index.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+such as:
+sp =
+1
+np
+np
+∑
+i = 1
+be (1.6)
+where np is the number of points assigned to cluster p during the initiation step
+by KSC.
```
Comment (paulodhiambo) [inline content/articles/introduction-to-kernel-methods-in-machine-learning/Index.md]: Format this section to make the equation readable.

**Comment by Kayere** [comment]
Hello @Brian3811,

Upon running your article through our 3rd party plagiarism checker it seemed to raise a few flags and the % was higher than we typically accept.

Please see attached PDF - and revisit the article to ensure we are contributing wholly unique and original content.

Be sure to see our resources page to see more info on plagiarism and what is considered as such.

[Introduction to Kernel Methods in Machine Learning #5365 Copyleaks Report.pdf](https://github.com/section-engineering-education/engineering-education/files/7692635/Introduction.to.Kernel.Methods.in.Machine.Learning.5365.Copyleaks.Report.pdf)

**Pull request closed without merging**